src/include/duckdb/storage/statistics/base_statistics.hpp

```diff
@@ -39,6 +39,8 @@ class BaseStatistics {
 	unique_ptr<BaseStatistics> validity_stats;
 	//! The approximate count distinct stats of the column (if any)
 	unique_ptr<BaseStatistics> distinct_stats;
+	idx_t distinct_count; // estimate that one may have even if distinct_stats==nullptr
```

Comment:
Could we move the `distinct_count` to the `DistinctStatistics` instead? Then the `DistinctStatistics` could have either a HLL or a count.